components/ai_chat/content/browser/ai_chat_tab_helper.cc

```diff
@@ -17,6 +17,7 @@
 #include "base/notreached.h"
 #include "base/ranges/algorithm.h"
 #include "base/strings/string_util.h"
+#include "brave/common/brave_channel_info.h"
```

Comment:
this is a layering violation, you need to pass the channel into the TabHelper

Reply:
Fixed.